portal-terminal/Makefile

```
@@ -0,0 +1,51 @@
+.PHONY: help install build dev test lint format clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)